src/js/embeds.js

```diff
@@ -294,8 +294,7 @@
         }
         var pastedUrl = e.originalEvent.clipboardData.getData('text');
 
-        if (regexHelper.vimeo_parser(pastedUrl) ||
-            regexHelper.youtube_parser(pastedUrl)) {
+        if (regexHelper.urlParser(pastedUrl)) {
```

Comment:
Do we really need to test it this thoroughly? Wouldn't it be enough to test if `pastedUrl` starts with http, https, or // - `/^(http(s?):)?\/\//`? And the rest would be handled by oEmbed or out embed function.

This way we wouldn't have to add new file - `regexHelper.js` and regexp wouldn't look so scary. I just want to keep it simple.